sdk/go/common/util/gitutil/git.go

```diff
@@ -54,7 +54,7 @@ const (
 
 	// GitLabHostName The host name for GitLab.
 	GitLabHostName VCSKind = "gitlab.com"
-	// GitHubHostName The host name for GitHub.
+	// GitHubHostName The host name for GitHub. This is not the full hostname to account for GitHub Enterprise.
```

Comment:
Either revert this, or add the same comment above for GitLab, otherwise this is confusing.

Reply:
Reverted it, since we aren't using the Constant anymore